modules/dataquery/jsx/react.filterBuilder.js

```diff
@@ -153,9 +161,11 @@ var FilterRule = React.createClass({
 		        };
 		    switch(rule.operator) {
 		    	case "equal":
+		    	case "isNull":
 		    		ajaxRetrieve("queryEqual.php");
 		    		break;
```

Comment:
This logic doesn't seem right.. why is isNull equivalent to equal and isNotNull equivalent to notEqual?